docs/admin/components/editor.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Editor components
+---
+
+import { PropsTable, PropsTableRow } from "../../../src/components/propsTable";
+import * as SharedProps from "../../../src/components/props";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Block editor
+
+The `BlockEditor` component is the main component of the editor. It is responsible for rendering the editor's toolbar and the editor's content.
```

Comment:
I would add here example of api definition. Also how to render it on front-end.

Reply:
I'll move documentation of the Block editor to separate pull request.